MAINTAINERS.yml

```diff
@@ -1101,6 +1101,7 @@ Release Notes:
     - doc/hardware/peripherals/gnss.rst
     - drivers/gnss/
     - include/zephyr/drivers/gnss.h
+    - tests/drivers/build_all/input/
```

Comment:
This should probably be tests/drivers/build_all/gnss/ :)

Reply:
just when I thought I was on the top of my copy -> paste -> rename game, dammit.